tests/manual/linkdecorator.js

```diff
@@ -21,30 +21,30 @@ ClassicEditor
 		plugins: [ Link, Typing, Paragraph, Clipboard, Undo, Enter ],
 		toolbar: [ 'link', 'undo', 'redo' ],
 		link: {
-			decorators: [
-				{
+			decorators: {
+				IxExternal: {
```

Comment:
`is`. Also most likely `isExternal` because we use camel case for props in the project.